qml/Components/PhysicalKeysMapper.qml

```diff
@@ -45,13 +45,21 @@ Item {
     readonly property bool superPressed: d.superPressed
     readonly property bool superTabPressed: d.superTabPressed
 
+    // Add a timeout for screenshots, but let it be half of normal long press
+    readonly property int screenshotTimeout: mouseArea.pressAndHoldInterval / 2
```

Comment:
You don't need the MouseArea, you can use `Qt.styleHints.mousePressAndHoldInterval`